acceptance/cmd/psql/script

```diff
@@ -0,0 +1,26 @@
+mv echo-arguments.sh psql
+chmod +x psql
```

Comment:
nit: Setting +x on `echo-arguments.sh` should also work. The executable bit for files is stored in git.